core/src/test/java/org/elasticsearch/test/ESTestCase.java

```diff
@@ -357,9 +385,16 @@ public static String randomRealisticUnicodeOfCodepointLength(int codePoints) {
         return array;
     }
 
+    public static String randomTimeValue(int lower, int upper) {
```

Comment:
Not sure what worth having a lower and upper limit is if it's randomly seconds versus days?